superset/connectors/sqla/models.py

```diff
@@ -537,16 +537,9 @@ def select_star(self) -> str:
             latest_partition=False,
         )
 
-    def get_col(self, col_name: str) -> Optional[Column]:
-        columns = self.columns
-        for col in columns:
-            if col_name == col.column_name:
-                return col
-        return None
-
```

Comment:
This method was functionally an identical copy of `BaseDatasource.get_column` (the parent class), hence was removed and references updated accordingly.